### PR TITLE
change file

### PR DESCRIPTION
**Pull request by mansurmabo**

### CODE REVIEW

**Review by Zombasik — APPROVED** [review]

**Review by usercoderr — APPROVED** [review]

**Review by sulaimanovaNad — APPROVED** [review]

**Review by AktanK13 — APPROVED** [review]

**Review by fymour — CHANGES_REQUESTED** [review]
peredelai

**Review by Al1han — APPROVED** [review]

**Review by Gulbaira — APPROVED** [review]

File: Nadira.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
 </head>
 <body>
     <h1 style="color: red;">!!!Aloha!!!</h1>
+    <h2 style="color: rgb(4, 0, 255);">!!!Hello Nadira!!!</h2>
```
Comment (AktanK13) [inline Nadira.html:11]: fix it Nadira!!!

Comment (mansurmabo) [reply Nadira.html:11]: won't do it

**Review by rasl-rgb — APPROVED** [review]

File: Nadira.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
...
     <p style="color: red;">!!!hi!!!</p>
```
Comment (TurgunJS) [inline Nadira.html:12]: what is it

**Pull request merged**